test/ui/defaultHtmlStepUi.spec.js

```diff
@@ -15,6 +15,16 @@ describe('Sequencer step HTML', function() {
   };
```

Comment:
Can we keep this under the directory structure test/ui/spec/filename.spec.js as the jasmine docs suggest to put jasmine tests under spec folder.

Reply:
done